### PR TITLE
NIAD-157 dockerise nhais

### PR DESCRIPTION
**Pull request by mattd-kainos**

### CODE REVIEW

**Review by Broom94 — DISMISSED** [review]

File: nhais/Dockerfile.dynamodb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+RUN yum install --assumeyes --quiet python3 && pip3 install -q awscli
+
+USER dynamodblocal
+
+# Create a table called mhs_state with primary key "key"
+RUN nohup bash -c "java -jar DynamoDBLocal.jar -sharedDb &" && \
+         AWS_ACCESS_KEY_ID=test AWS_SECRET_ACCESS_KEY=test aws dynamodb \
+        --endpoint-url http://localhost:8000 \
+        --region eu-west-2 create-table \
+        --table-name transaction_id_counter \
```
Comment (Alicja-Kainos) [inline nhais/Dockerfile.dynamodb]: 1) This table should be renamed to 'generated_id_conter'
2) I would need to add a nhais_state table in the task NIAD-155. Should I create a similar file and then we will merge it, or should I just prepare code that will have placeholder for the table and add it after you'll merge this pr, or should I create branch from this branch to have your changes?

**Review by Alicja-Kainos — CHANGES_REQUESTED** [review]
rename the table 'transaction_id_counter' into 'generate_id_counter'

File: nhais/Dockerfile.dynamodb (hunk 1 above)
Comment (mattd-kainos) [reply nhais/Dockerfile.dynamodb]: I do still need to remove this table from the MHS dockerfile!

File: dockers/common/dynamodb-local/Dockerfile [hunk 2]
```diff
@@ -21,13 +21,6 @@ RUN nohup bash -c "java -jar DynamoDBLocal.jar -sharedDb &" && \
         --table-name sync_async_state \
         --attribute-definitions AttributeName=key,AttributeType=S \
         --key-schema AttributeName=key,KeyType=HASH \
-        --provisioned-throughput ReadCapacityUnits=1,WriteCapacityUnits=1 && \
```
Comment (mattd-kainos) [inline dockers/common/dynamodb-local/Dockerfile:24]: Should release a new nhsdev/nia-dynamodb-local now that this has been removed

**Review by Alicja-Kainos — APPROVED** [review]

**Review by Broom94 — APPROVED** [review]

**Pull request merged**